src/_P087_SerialProxy.ino

```diff
@@ -280,6 +325,20 @@ void P087_html_show_matchForms(struct EventStruct *event) {
     static_cast<P087_data_struct *>(getPluginTaskData(event->TaskIndex));
 
   if ((nullptr != P087_data)) {
+
+    addFormNumericBox(F("Read HEX Data Length"), getPluginCustomArgName(P087_HEX_DATA_LEN_POS), P087_data->getHEXDataLength(), -1024, 1024);
+    addFormNote(F("HEX = 0 Read ASCII 32 - 217 and end read on [Enter]; HEX > 0 Read N Length."));
+    addFormNote(F("HEX < 0 ,Read header next N data get Length."));
+    addFormNote(F(" e.g: set -2 , Read header next 2 data, [F11FE22EB66BA88A][00][08] , -1 = [00],-2 = [08], Set Length 8"));
```

Comment:
Not sure if this rather descriptive note should be here, or in the (missing) documentation.